tests/bindings/python/test_importer.py

```diff
@@ -147,6 +147,39 @@ def test_clear_imports(self):
         importer.clearImports(model)
         self.assertTrue(model.hasUnresolvedImports())
 
+    def test_is_resolved_units(self):
+        from libcellml import Importer, Parser
+
+        parser = Parser()
+        importer = Importer()
+
+        model = parser.parseModel(file_contents("importer/master_units.cellml"))
+        self.assertEqual(0, parser.issueCount())
+
+        importer.resolveImports(model, resource_path("importer/"))
+
+        self.assertEqual(0, importer.issueCount())
+
+        u = model.units(0)
+
+        self.assertTrue(u.isResolved())
+
+    def test_is_resolved_component(self):
```

Comment:
I know we test `isResolved()`, but because we actually test that it returns `False`, maybe the test should be called `test_is_not_resolved_component()`?

Reply:
I'm happy with calling it after the method being tested.

Reply:
Ok, so I shall remain confused when reading the name of that test.